src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -341,11 +278,14 @@ declare global {
     protected _onMovementFrame(
       dt: unknown,
       anim: Array<{ context: unknown; name: string | null; duration: number; ontick: unknown }>,
```

Comment:
improve, or add a TODO. This seems to be closely related to `CanvasAnimation.animateLinear` and `anvasAnimation.Attribute`